internal/impl/validation/validator.go

```diff
@@ -144,3 +144,17 @@ func BooleanPropertyValidation(key string, property interface{}) (bool, bool, er
 	}
 	return ret, true, nil
 }
+
+// BooleanPropertyTruthyValidation function
+func BooleanPropertyTruthyValidation(key string, property interface{}) (bool, bool, error) {
```

Comment:
As per the internal discussion. We'll keep the endpoint durability property with setter. But then we must defer to ccsmp for the validity of the property value. This only test golang needs to do is for whether the value is boolean or not. Why was this added?